File: CyberHunters/Assets/Scripts/spectrum/TeleportationLevel.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class TeleportationLevel : MonoBehaviour
{
    public int indexLevel;

    void OnTriggerEnter2D (Collider2D other)
    {
        if (other.CompareTag("Player"))
        {
            SceneManager.LoadScene(indexLevel);
        }
        
    }
}

File: Java/Assets/_SPECTRUM/scripts/TeleportationLevel.java
import unityengine.*;
import unityengine.SceneManagement.*;
import java.util.*;

public class TeleportationLevel extends MonoBehaviour
{
	public int indexLevel;
	public Animator levelAnim;
	public Animator musicAnim;
	public float WaitTime;

	public final void Activate()
	{
		StartCoroutine(Wait());
		levelAnim.SetTrigger("FadeOut");
		DialogManager.isTalking = false;
	}

	private Iterator Wait()
	{
		musicAnim.SetTrigger("fadeOut");
//C# TO JAVA CONVERTER TODO TASK: Java does not have an equivalent to the C# 'yield' keyword:
		yield return new WaitForSeconds(WaitTime);
		SceneManager.LoadScene(indexLevel);
	}


}